### ISSUE TITLE
Fleshing out appmetadata()

### ISSUE THREAD

**Issue by marcverhagen** [clamsproject/clams-python#19]
The `appmetadata()` method always does the same as long as `__init__()` or some other method sets the metadata instance variable. To simplify wrapping we could just define it on `ClamsApp`:

```python
def appmetadata(self):
    return json.dumps(self.metadata)
```

**Comment by keighrim** [comment]
8d25e3a implements this behavior, but as it changes the ABS API spec, it breaks all existing apps (we're updating them anyway so I think it doesn't matter).

**Comment by keighrim** [comment]
closed via #22
